src/v/kafka/protocol/schemata/generator.py

```diff
@@ -42,6 +35,9 @@
 import jsonschema
 import jinja2
 
+sys.path.append(sys.argv[1])
+from flex_versions import flex_versions
```

Comment:
Another struggle with pex. I want to be able to import the generated python file created as output of the python program created in the previous change.

Reply:
Not needed